guides/ecto.md

```diff
@@ -2,23 +2,27 @@
 
 > **Requirement**: This guide expects that you have gone through the [introductory guides](installation.html) and got a Phoenix application [up and running](up_and_running.html).
 
-Most web applications today need some form of data validation and persistence. In the Elixir ecosystem, we have Ecto to enable this. Before we jump into building database-backed web features, we're going to focus on the finer details of Ecto to give a solid base to build our web features on top of. Let's get started!
+Most web applications today need some form of data validation and persistence. In the Elixir ecosystem, we have [`Ecto`](https://hexdocs.pm/ecto/) to enable this. Before we jump into building database-backed web features, we're going to focus on the finer details of Ecto to give a solid base to build our web features on top of. Let's get started!
 
 Phoenix uses Ecto to provide builtin support to the following databases:
 
-* PostgreSQL (via [`postgrex`](https://github.com/elixir-ecto/postgrex))
-* MySQL (via [`myxql`](https://github.com/elixir-ecto/myxql))
-* MSSQL (via [`tds`](https://github.com/livehelpnow/tds))
+* PostgreSQL (via [`postgrex`](https://hexdocs.pm/postgrex/))
+* MySQL (via [`myxql`](https://hexdocs.pm/myxql/))
+* MSSQL (via [`tds`](https://hexdocs.pm/tds/))
```

Comment:
Here I am not 100% convinced it is best to link to the docs.

Reply:
It is not, The github projects is usually the ones with installation instructions. Please revert. :)